tests/e2e/scenarios/test_reborn_webui_v2_legacy_skills.py

```
@@ -0,0 +1,247 @@
+        await expect(editor).to_be_hidden(timeout=5000)
+
+        assert len(harness["update_requests"]) == 1
+        update = harness["update_requests"][0]
+        assert update["name"] == "markdown-helper"
+        assert update["headers"].get("x-confirm-action") == "true"
+        assert "Updated E2E skill" in update["body"]["content"]
+
+        loop = asyncio.get_running_loop()
+        dialog_future = loop.create_future()
+
+        def handle_dialog(dialog):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `handle_dialog`

Add return type annotation: `None`

(ANN202)